src/ape/types/signatures.py

```diff
@@ -22,6 +23,7 @@ class _Signature:
     v: int
     r: bytes
     s: bytes
+    messageHash: Optional[HexBytes] = None
```

Comment:
suggestion: I suspect setting the type as `bytes` here may fix your tests.  You can still set the value as `HexBytes` since it's a derivative of `bytes` and pydantic will serialize it as such.

Reply:
done